pkg/rid/store/cockroach/store.go

```diff
@@ -71,14 +72,16 @@ func (s *Store) Transact(ctx context.Context, f func(repo repos.Repository) erro
 	// "store" for everything
 	ctx, cancel := context.WithTimeout(ctx, DefaultTimeout)
 	defer cancel()
+
+	storeVersion, err := s.GetVersion(ctx)
```

Comment:
Querying the version every transaction seems like the same optimization opportunity as for interactions.

Reply:
Never mind; we would need to do this to determine when the database was upgraded to 3.1.0.